src/screens/ContactScreen/index.js

```diff
@@ -212,11 +228,13 @@ const ContactScreen = ({navigation}) => {
         // onPress={handleSearch}
       />
 
-      <View>
-        {selectedUsers && (
-          <ContactPreview users={selectedUsers} onPress={(user) => handleSelected(user)} />
-        )}
-      </View>
+      {sourceScreen !== ANONYMOUS && (
+        <View>
+          {selectedUsers && (
+            <ContactPreview users={selectedUsers} onPress={(user) => handleSelected(user)} />
+          )}
+        </View>
+      )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

A conditional rendering block is added to only render the `ContactPreview` component if the source screen is not `ANONYMOUS`. This change prevents the display of selected users in anonymous chats, which is consistent with the behavior defined in the `handleSelected` function.
```diff
+ {sourceScreen !== ANONYMOUS && (
+   <View>
+     {selectedUsers && (
+       <ContactPreview users={selectedUsers} onPress={(user) => handleSelected(user)} />
+     )}
+   </View>
+ )}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->